certbot/cli.py

```diff
@@ -1051,9 +1058,7 @@ def _paths_parser(helpful):
         verb = helpful.help_arg
 
     cph = "Path to where cert is saved (with auth --csr), installed from or revoked."
```

Comment:
Is that a lack of oxford comma I see? Probably because other parts are hogging all the commas.